src/vue/common/fees/fees.record.js

```diff
@@ -0,0 +1,47 @@
+import { FEE_TYPES } from '@tokend/js-sdk'
+export class FeesRecord {
+  constructor ({ fixed = '', calculatedPercent = '', assetCode, type }) {
+    this.fixed = fixed
+    this.calculatedPercent = calculatedPercent
+    this.assetCode = assetCode
+    this.type = type
+  }
+}
+
+export class CoupledFeesRecord {
+  constructor ({ destination, source, assetCode }) {
+    if (
+      !(destination instanceof FeesRecord) ||
+      !(source instanceof FeesRecord)
+    ) {
+      throw new Error('One of the input objects is not FeesRecord')
+    }
+    this.destination = destination
+    this.source = source
+    this.assetCode = assetCode
+    this.IsAnyExternalFee = false
+  }
+
+  get isAnyDestinationFee () {
+    return Boolean(+this.destination.calculatedPercent) ||
+      Boolean(+this.destination.fixed)
+  }
+  get isAnySourceFee () {
+    return !!this.source.calculatedPercent ||
+    !!this.source.fixed
+  }
+  get isWithdrawable () {
+    return this.type === FEE_TYPES.withdrawalFee
+  }
+  get getIsAnyExternalFee () {
+    return this.IsAnyExternalFee
+  }
+  set setIsAnyExternalFee (opts) {
```

Comment:
ok, you got the setter idea wrong. pls remove the setter. the `isAnyExternalFee` should be set like this:
```js
  // ...
  someCoupledFeeInstance.isAnyExternalFee = true
  // ...
```
it should not do any calculations